app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -11,7 +11,7 @@
 /**
  * Helper method for the logical functionality of the {@link CreateProfileActivity}.
  */
-class CreateProfileHelper {
+final class CreateProfileHelper {
```

Comment:
We've no reason to not make it final. If we want to extend the class in the future, then you we can always use a decorator (good coding style anyway). Otherwise we'll have to eliminate the class's self-use of overridable methods entirely and well... That costs a lot of time. Additionally, when you decide to make a class extendable, you should actually also test it, which costs even more time.